### DISCUSSION TITLE
Display space for Switches and Patchpanels like Racks

### DISCUSSION THREAD

**Discussion by nbrgit** [netbox-community/netbox#13468] [General]
Hey All,
it would be helpful, if you can see how mutch ports are in use or free, like the available Space in a Rack.

Is there some possibility to display that?

Thank you!

**Comment by deku-m** [comment]
Someone created issue and also a plugin but too bad i havent seen it is released or included in netbox itself:
https://github.com/netbox-community/netbox/issues/11633

So not yet.
